lib/sles4sap_publiccloud.pm

```diff
@@ -608,4 +610,63 @@ sub delete_network_peering {
     }
 }
 
+=head2 get_hana_database_info
+
+    Parses "hdbsql" command output.
+    Returns 1 if hana database is online otherwise 0 offline.
+
+=over 1
+
+=item B<RESULT> - default "-1" representing empty value.
+
+=back
+=cut
+
+sub get_hana_database_info {
+    my ($self) = @_;
+    my $result = -1;
+    $self->is_hana_online();
+    my $output_cmd = $self->run_cmd(cmd => "hdbsql -u SYSTEM -p Linux1234 -i 00 'SELECT * FROM SYS.M_DATABASES;'", runas => get_required_var("SAP_SIDADM"));
+    my $hana_database_online = '"HDB","HDB-00","YES","ACTIVE","","","DEFAULT","?"';
```

Comment:
I'm not 100% sure, but I think the second element of this string (`"HDB-00"`) is related to the SID and the Instance Number. Meaning that if you run the test with different values for INSTANCE_SID and INSTANCE_ID, then it will not match even when it's supposed to.

Reply:
This will be check in a VR